apps/portals/src/AppInitializer.tsx

```diff
@@ -56,11 +56,15 @@ function useSession(
   )
 
   const initAnonymousUserState = useCallback(async () => {
-    await SynapseClient.signOut()
     // reset token and user profile
     setToken(undefined)
     setUserProfile(undefined)
     setHasCalledGetSession(true)
+    try {
+      await SynapseClient.signOut()
+    } catch (e) {
+      console.error('Unable to sign out: ', e)
+    }
```

Comment:
We might want to add general fallback logic if we hit a CORS issue when we hit the servlet. Not necessary to merge this though

Reply:
The current fallback logic is found in getSession(), where it calls this function (initAnonymousUserState()) if getAccessTokenFromCookie() fails.